kie-drools-wb-parent/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/KieDroolsWorkbenchEntryPoint.java

```diff
@@ -154,12 +154,18 @@ private void initNavTreeEditor() {
 
         // Due to a limitation in the Menus API the number of levels in the workbench's menu bar
         // navigation tree node must be limited to 2 (see https://issues.jboss.org/browse/GUVNOR-2992)
-        navTreeEditor.setMaxLevels(NavTreeDefinitions.GROUP_WORKBENCH,
-                                   2);
+        navTreeEditor.setMaxLevels(NavTreeDefinitions.GROUP_WORKBENCH, 2);
```

Comment:
Your formatting is right?

Reply:
@ederign is there any general rule for breaking s single line?

Anyway, feel free to fix it as you consider :-)

Reply:
I think the most recent code style allow parameters at the same line.